### ISSUE TITLE
Detail using specific components in an MDXProvider

### ISSUE THREAD

**Issue by spences10** [PaulieScanlon/mdx-embed#81]
In the docs:

```jsx
import React from 'react';
import { MDXEmbedProvider } from 'mdx-embed';

export default (props) => <MDXEmbedProvider>{props.children}</MDXEmbedProvider>;
```

Say if the user only wants to use the the `<CodePen />` and `<Gist />` components, they're not tied into using the `MDXEmbedProvider` so I think it should be made clear, something along the lines of:

Only want to use a couple of components from MDX Embed, you can import only the components you need into the `MDXProvider`

```jsx
import React from 'react';
import { MDXProvider } from '@mdx-js/react'
import { CodePen, Gist } from 'mdx-embed';

const components = {
  CodePen,
  Gist
}

export default (props) => <MDXProvider components={components}>{props.children}</MDXProvider>;
```

**Comment by PaulieScanlon** [comment]
@spences10 yeah lovely idea mate!

**Comment by PaulieScanlon** [comment]
addressed in [#90]
